dev-docs/source/ReleaseChecklist.rst

```diff
@@ -15,260 +15,327 @@ Roles
+| | **Wednesday**,                |                           | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Release Manager <release-manager-checklist>`                     |
+| | 1 week & 3 days               |                           | Maintenance Tasks & Next release development  |                                                                          |
++---------------------------------+---------------------------+-----------------------------------------------+--------------------------------------------------------------------------+
+| | **Tuesday**,                  | Beta Testing Ends         | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Quality Assurance Manager <quality-assurance-manager-checklist>` |
+| | 4 days                        |                           | Maintenance Tasks & Next release development  | | :ref:`Release Editor <release-editor-checklist>`                       |
+|                                 |                           |                                               | | :ref:`Release Manager <release-manager-checklist>`                     |
++---------------------------------+---------------------------+-----------------------------------------------+--------------------------------------------------------------------------+
+| | **Friday**,                   | Release Eve               | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Release Manager <release-manager-checklist>`                     |
+| | 1 day                         |                           | Maintenance Tasks & Next release development  |                                                                          |
++---------------------------------+---------------------------+-----------------------------------------------+--------------------------------------------------------------------------+
+| | **Monday**,                   | Release Day               | Blocker bug fixes, Testing, Release Notes,    | | :ref:`Release Manager <release-manager-checklist>`                     |
+| | Release Day                   |                           | Maintenance Tasks & Next release development  | | :ref:`Technical Release Manager <technical-release-manager-checklist>` |
++---------------------------------+---------------------------+-----------------------------------------------+--------------------------------------------------------------------------+
 
 .. note::
-   We used to release on a Friday, but changed to a Monday as no support was available over the weekend in case issues
-   were raised with the newly released version.
-
-Friday, Release-4 weeks (one week before code freeze)
-#########################################################
+   We used to release on a Friday, but changed to a Monday as support was not available 
```

Comment:
I think we can remove this note now. This document isn't supposed to serve as a historical record; that's what git is for.